src/Components/category-card-one/category-card-one.tsx

```diff
@@ -0,0 +1,44 @@
+import Link from 'next/link';
+import React from 'react';
+
+interface CategoryCardOneProps {
+	title: string;
+	nop: number;
+	icon: React.JSX.ElementType;
+	href?: string;
+	asDiv?: boolean;
+	onClickAction?: () => void;
+}
+
+const componentTypeDecider = (
+	title: string,
+	nop: number,
```

Comment:
Don't abbreviate unnecessarily.
When I first look at `nop` --> _What the hell is nop?_
If I saw `numberOfProducts` instead --> _Oh this prop is for number of products belong to this category_

Reply:
Noted
We will make changes to that as well